lib/ioki/model/passenger/user.rb

```diff
@@ -27,7 +27,6 @@ class User < Base
         attribute :registered, on: :read, type: :boolean
         attribute :remaining_referrals, on: :read, type: :integer
         attribute :unique_customer_id, on: :read, type: :string
-        attribute :logpay_customer_set, on: :read, type: :boolean
```

Comment:
@eikes We still need this attribute in an app. It is also in the API documentation.

Did you remove it because it's not in the OpenAPI spec? If so, we probably should add it there.

Reply:
I removed it here because it is defined twice. All the removals in this PR are because the same attribute was defined twice in the model. It's still there. See for yourself: https://github.com/ioki-mobility/ioki-ruby/blob/e2fe57720ee07bbc32ade1174b6ea99696ac6ad6/lib/ioki/model/passenger/user.rb

Reply:
Oh, so I added it despite it being there already :thinking: Sorry for the noise!